spot-client/src/zoom/sdk/sdk.js

```diff
@@ -177,6 +192,21 @@ class Sdk {
         this._videoMuteController.setMute(mute);
     }
 
+    /**
+     * Tries to rejoin the meeting but with a password.
+     *
+     * @param {string} passWord - The password to use for joining the meeting.
+     * @returns {void}
+     */
+    submitPassword(passWord) {
+        this._cachedJoinOptions = {
```

Comment:
this might be not necessary as `this.joinMeeting` will overwrite it, so maybe just pass the new options?